src/server/middlewares/redirects.ts

```diff
@@ -26,20 +27,16 @@ export const forceHost = ({
 
 export const startPageRedirect: IMiddleware<object> = async (ctx) => {
   const actualIp = ctx.ip
-  const country = lookupCountry(actualIp)
+  const fallbackCountryLabel = fallbackLocale.label
+  const countryLabel =
+    lookupCountry(actualIp)?.toLowerCase() || fallbackCountryLabel
   const queryStringMaybe = ctx.querystring ? '?' + ctx.querystring : ''
 
-  if (country === 'NO') {
-    ctx.redirect(`/no${queryStringMaybe}`)
-    return
+  if (Object.keys(locales).includes(countryLabel)) {
+    ctx.redirect(`/${countryLabel}${queryStringMaybe}`)
+  } else {
+    ctx.redirect(`/${fallbackCountryLabel}${queryStringMaybe}`)
   }
```

Comment:
Maybe just have a early return? :) 
```suggestion
  if (Object.keys(locales).includes(countryLabel)) {
    ctx.redirect(`/${countryLabel}${queryStringMaybe}`)
    return
  }
  
  ctx.redirect(`/${fallbackCountryLabel}${queryStringMaybe}`)

```

Reply:
Sure, I don't have very strong opinions on using the `else` 🙃